lib/App/Sqitch/ItemFormatter.pm

```diff
@@ -110,8 +105,8 @@ has formatter => (
                 c => sub {
                     return "$_[0]->{committer_name} <$_[0]->{committer_email}>"
                         unless defined $_[1];
-                    return $_[0]->{committer_name}  if $_[1] ~~ [qw(n name)];
-                    return $_[0]->{committer_email} if $_[1] ~~ [qw(e email)];
+                    return $_[0]->{committer_name}  if $_[1] =~ /^n(?:ame)?/;
```

Comment:
Technically `$` isn't quite the same since it allows for an optional newline at the end of the string, so it'd match the literal `"name\n"`. To actually match the end of the string you need `\z`. In that case you might also want to use `\A` at the beginning for symmetry, but without the `/m` flag this isn't necessary.

Reply:
Can't believe I forgot the `$`. Changed to `\A` and `\z` in 46b8d78.